source/GameData.cpp

```diff
@@ -990,7 +992,22 @@ void GameData::PrintWeaponTable()
 		
 		cout << outfit.Homing() << '\t';
 		double strength = outfit.MissileStrength() + outfit.AntiMissile();
-		cout << strength << '\n';
+		cout << strength << '\t';
+		
+		double turretTurn = outfit.TurretTurn() * 60.;
+		cout << turretTurn << '\t';
+		
+		double heatDmg = outfit.HeatDamage() * 60. / outfit.Reload();
+		cout << heatDmg << '\t';
+		double ion = outfit.IonDamage() * 6000. / outfit.Reload();
+		cout << ion << '\t';
+		double disruption = outfit.DisruptionDamage() * 6000. / outfit.Reload();
+		cout << disruption << '\t';
+		double slowing = outfit.SlowingDamage() * 6000. / outfit.Reload();
+		cout << slowing << '\t';
+		
+		double piercing = outfit.Piercing() * 100.;
+		cout << piercing << '\n';
```

Comment:
It'd make more sense to just leave piercing as a value between 0 and 1. (If you have a "0.1" in a spreadsheet and you format that cell as a percentage, it'll correctly display as "10%." And if you want to do math with that value, you don't need to divide it by 100 first.)